bin/index.js

```diff
@@ -236,6 +257,60 @@ function getTruffleArtifact(artifactsDir, fileName) {
     return contract;
 }
 
+function watchBrownieArtifacts(dir, projectConfig) {
+    if (!dir) {
+        console.log('Please specify a directory to watch.');
+        return;
+    }
+    // Might need to add a way to specify which deployment folder to watch
+    const artifactsDir = path.format({
+        dir: dir,
+        base: "build/deployments"
+    }); 
+    const watcher = chokidar.watch('.', { cwd: artifactsDir })
```

Comment:
Replace with `const watcher = chokidar.watch('./**/*.json', { cwd: artifactsDir, ignored: 'map.json' })`. I somehow had a non json file in the folder that caused an error, and `map.json` shouldn't be useful to parse.

Reply:
done!